src/libs/actions/Session/index.js

```diff
@@ -72,24 +74,33 @@ function createAccount(login) {
  */
 function signOut() {
     Log.info('Flushing logs before signing out', true, {}, true);
+
     if (credentials && credentials.autoGeneratedLogin) {
```

Comment:
NAB, just noting - it's kind of unclear why we are only calling "LogOut" when we have an `autoGeneratedLogin`. It made more sense before to see "DeleteLogin".

Reply:
Hmm is your point that it feels kinda weird to pass `login` data to a `LogOut` command?

Reply:
I don't disagree, but the user doesn't know they're deleting a login, they just know they're signing out, which is why we're calling the command "LogOut"

Reply:
Not quite. My point was that the code was easier to follow before because you see:

- a check for the credentials
- delete credentials because they exist

now we see 

- a check for the credentials
- who knows what happens next or why the check even exists

Reply:
Cool sounds like this is also covered in your below comment, resolving as we discussed this & I removed this check